src/components/Form/__tests__/Checkbox.spec.js

```diff
@@ -18,8 +18,8 @@ describe('<Checkbox />', () => {
     mounted = mount(<Checkbox {...props} />);
   });
 
-  it('renders the id', () => {
-    expect(mounted.find(`#${props.id}`).length).toEqual(1);
+  it('renders the component', () => {
+    expect(mounted.find(Checkbox).length).toEqual(1);
```

Comment:
Why does this not cover the original spec anymore?

Reply:
will add the check for the id again 👍